en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4191 @@
+<td>
+
+The start date of the time range. The date specified here must be a date earlier than the date specified via the
+<code>to</code> parameter. If the defined date and time entry do not have a timezone, the <code>UTC</code> time zone (z) is used. It is
+required to specify a value for this parameter.
+e.g., <code>'2021-08-16T12:00:00.000+05:30'</code>.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>to</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+The end date of the time range. The date specified here must be a date later than the date specified via the <code>from</code>
+parameter. If the defined date and time entry do not have a timezone, the <code>UTC</code> time zone (z) is used. It is
+required to specify a value for this parameter.
+e.g., <code>'2021-08-16T12:30:00.000+05:30'</code>.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### TopAPIsByAlertCountFilter
+
+Filters the top APIs by alert count results.
+
+<table>
+<thead>
+<tr>
+<th colspan="2" align="left">Field</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>paginationFilter</strong></td>
+<td valign="top"><a href="#paginationfilter">PaginationFilter</a></td>
+<td>
+
+The pagination filter.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>searchFilter</strong></td>
+<td valign="top"><a href="#searchfilter">SearchFilter</a></td>
+<td>
+
+The search filter.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+## Enums
+
+### ErrorCodeType
+
+Represents an error code type.
+
+<table>
+<thead>
+<th align="left">Value</th>
+<th align="left">Description</th>
+</thead>
+<tbody>
+<tr>
+<td valign="top"><strong>_4XX</strong></td>
+<td>
+
+Client side errors.
+
+</td>
+</tr>
+<tr>
+<td valign="top"><strong>_5XX</strong></td>
+<td>
+
+Server side errors.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### ErrorType
+
+Represents an error type.
+
+<table>
+<thead>
+<th align="left">Value</th>
+<th align="left">Description</th>
+</thead>
+<tbody>
+<tr>
+<td valign="top"><strong>PROXY</strong></td>
+<td>
+
+Proxy-related errors.
+
+</td>
+</tr>
+<tr>
+<td valign="top"><strong>TARGET</strong></td>
+<td>
+
+Target-related errors.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+## Scalars
+
+### Boolean
+
+The <code>Boolean</code> scalar type represents <code>true</code> or <code>false</code>.
+
+### Float
+
+The <code>Float</code> scalar type represents signed double-precision fractional values as specified by [IEEE 754](https://en.wikipedia.org/wiki/IEEE_floating_point).
+
+### ID
+
+The <code>ID</code> scalar type represents a unique identifier, often used to refetch an object or as key for a cache. The ID type appears in a JSON response as a String; however, it is not intended to be human-readable. When expected as an input type, any string (such as <code>"4"</code>) or integer (such as <code>4</code>) input value will be accepted as an ID.
```

Comment:
Replace 'will be" with 'is".

Reply:
This is an auto-generated description according to the graphql spec. Let's keep this as it is.